### ISSUE TITLE
sudo startunity won't work

### ISSUE THREAD

**Issue by sidharthsiddapureddy** [dnschneid/crouton#2601]
``````
Please paste the output of the following command here: sudo edit-chroot -all
```name: precise
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/precise...
crouton: version 1-20160518095657~master:fd37bb79
release: precise
architecture: amd64
xmethod: xorg
targets: unity
host: version 7978.74.0 (Official Build) stable-channel falco 
kernel: Linux localhost 3.8.11 #1 SMP Thu May 12 00:55:57 PDT 2016 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Not unmounting /mnt/stateful_partition/crouton/chroots/precise as another instance is using it.

#### Please describe your issue:
When I type in sudo startunity after I reboot my computer I get this error.
chronos@localhost ~/Downloads $ sudo startunity
Entering /mnt/stateful_partition/crouton/chroots/precise...

_XSERVTransmkdir: Owner of /tmp/.X11-unix should be set to root

X.Org X Server 1.11.3
Release Date: 2011-12-16
X Protocol Version 11, Revision 0
Build Operating System: Linux 2.6.42-61-generic x86_64 Ubuntu
Current Operating System: Linux localhost 3.8.11 #1 SMP Thu May 12 00:55:57 PDT 2016 x86_64
Kernel command line: cros_secure console= loglevel=7 init=/sbin/init cros_secure oops=panic panic=-1 root=/dev/dm-0 rootwait ro dm_verity.error_behavior=3 dm_verity.max_bios=-1 dm_verity.dev_wait=1 dm="1 vroot none ro 1,0 2506752 verity payload=PARTUUID=aa84daac-0b9a-2740-a037-b84094d818fc/PARTNROFF=1 hashtree=PARTUUID=aa84daac-0b9a-2740-a037-b84094d818fc/PARTNROFF=1 hashstart=2506752 alg=sha1 root_hexdigest=6d5e4b5e42e62ae05cef04000955575c4e603d93 salt=72f18d9aa44cdd48030029703d09a264777f6201556275ed3a651921098a5735" noinitrd vt.global_cursor_default=0 kern_guid=aa84daac-0b9a-2740-a037-b84094d818fc add_efi_memmap boot=local noresume noswap i915.modeset=1 tpm_tis.force=1 tpm_tis.interrupts=0 nmi_watchdog=panic,lapic iTCO_vendor_support.vendorsupport=3  
Build Date: 12 February 2015  02:49:01PM
xorg-server 2:1.11.4-0ubuntu10.17 (For technical support please see http://www.ubuntu.com/support) 
Current version of pixman: 0.30.2
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(++) Log file: "/tmp/Xorg.crouton.1.log", Time: Thu May 26 16:21:05 2016
(==) Using system config directory "/usr/share/X11/xorg.conf.d"

Fatal server error:
no screens found

Please consult the The X.Org Foundation support 
         at http://wiki.x.org
 for help. 
Please also check the log file at "/tmp/Xorg.crouton.1.log" for additional information.

 ddxSigGiveUp: Closing log
Server terminated with error (1). Closing log file.
/usr/bin/xinit: giving up
/usr/bin/xinit: unable to connect to X server: No such file or directory
/usr/bin/xinit: server error
Not unmounting /mnt/stateful_partition/crouton/chroots/precise as another instance is using it.

#### If known, describe the steps to reproduce the issue:
``````

**Comment by niharxavier** [comment]
The same thing happened to me. I was using xfce instead of unity.

Try rebooting your computer.

I still hav found no soulution to it.

WIll tell if I find any!

**Comment by DennisLfromGA** [comment]
@sidharthsiddapureddy - Follow this guide for help: [Common issues and reporting](https://github.com/dnschneid/crouton/wiki/Common-issues-and-reporting)

**Comment by yooncheolc** [comment]
Try below:
sudo enter-chroot
sudo rm /dev/dri/card0
sudo mv /dev/dri/card1 /dev/dri/card0
exit

then do **sudo startunity**

**Comment by matty312** [comment]
Try Updating:
sudo sh ~/Downloads/crouton -n precise -u

**Comment by grigall** [comment]
I had to update crouton like matty312 mentioned, then create a new chroot:

sudo sh ~/Downloads/crouton -r precise -t unity,xiwi -n testchroot

After that, I ran this code to check if I'd accidentally created more than one chroot (which I had):

sudo edit-chroot -a

Then, I restarted my Chromebook, opened the terminal, and deleted the extra chroot:

sudo delete-chroot name_of_undesired_chroot

When I typed in the "sudo startunity" command, it worked like a champ!

P.S. I haven't tried running unity in xiwi (it just errors out and goes to the unity desktop) but I'll post an update if/when I do.
